bigbluebutton-tests/playwright/chat/chat.spec.js

```
@@ -1,91 +1,201 @@
+    await chat.initPages(page);
     await chat.autoConvertEmojiSendPrivateChat();
   });
 
-  test('Private chat disabled when user leaves meeting', async () => {
+  test('Private chat disabled when user leaves meeting', async ({ browser, context, page }) => {
+    const chat = new Chat(browser, context);
+    await chat.initPages(page);
     await chat.chatDisabledUserLeaves();
   });
+
+  test.describe.only('Message actions', () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)